connector_magento/models/product_template/importer.py

```diff
@@ -250,6 +250,7 @@ class ProductTemplateImportMapper(Component):
     _inherit = 'magento.product.product.import.mapper'
     _apply_on = ['magento.product.template']
 
+    direct = [('sku', 'magento_default_code')]
```

Comment:
This is a problem here - this member is already in use in the magento.product.product.import.mapper - from which this does inherit. So you should not overwrite it - just extend it.
I will correct this with a new commit after the merge

Reply:
Great thanks